custom/rch/forms.py

```diff
@@ -0,0 +1,111 @@
+from django.core.urlresolvers import reverse
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+from django.db.utils import ProgrammingError
+from django.conf import settings
+from crispy_forms import layout as crispy
+from crispy_forms.layout import Layout
+from crispy_forms.helper import FormHelper
+from crispy_forms.bootstrap import StrictButton
+from corehq.apps.hqwebapp import crispy as hqcrispy
+from custom.rch.models import AreaMapping
+
+
+def get_choices_for(value_field, display_field):
+    """
+    return a tuple of tuples with value and display text for each unique available option for
+    display field
+    for ex: return all combinations for state code and state name for all unique state names
+    available
+    :param value_field: column for value
+    :param display_field: column for display text
+    :return: tuple of tuples
+    """
+    if settings.UNIT_TESTING:
+        return tuple()
```

Comment:
This doesn't seem right. I don't see anything in this method that would prevent it from working in unit tests.

Reply:
The error here https://travis-ci.org/dimagi/commcare-hq/builds/279301373

Reply:
sorry, the correct build for the error is https://travis-ci.org/dimagi/commcare-hq/jobs/279305659